### PR TITLE
Santa - Fix part of lint issues

### PR DESCRIPTION
**Pull request by planarvoid**
* UnusedParent in layouts
* Missing ViewHolder in adapters
* Missing recycle()
* Attribute unused on older versions
* Inconsistent layouts
* Reference ID not in current layout
* Unnecessary divider copy

### CODE REVIEW

File: WordPress/lint.xml [hunk 1]
```diff
@@ -37,4 +37,8 @@
         <ignore path="**/generated/**" />
         <ignore path="**/strings.xml" />
     </issue>
+
+    <issue id="TypographyEllipsis" severity="warning">
+        <ignore regexp=".*values\-[a-z][a-z].*\/strings\.xml"/>
```
Comment (maxme) [inline WordPress/lint.xml]: Eventually, we can replace 3 dots `...` by the ellipsis `…` character in the[ script that pulls translations](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/tools/update-translations.sh). I'm not sure if there is any use case for the 3 dots in the strings file.

Comment (planarvoid) [reply WordPress/lint.xml]: I'll look into it! I think you're right

**Comment by malinajirka** [comment]
I've checked Vojta's changes and it looks good!

**Comment by malinajirka** [comment]
Looks good!;)

**Pull request merged**